src/lib/logger.ts

```diff
@@ -41,9 +42,6 @@ const loggerMiddleware = pinoHttp({
   },
   // Set to `false` to prevent standard serializers from being wrapped.
   wrapSerializers: false,
-  autoLogging: {
-    ignorePaths: [],
```

Comment:
`ignorePaths` is deprecated (and no longer appears in the type definitions) - but we weren't using it anyway!